lib/mirador_rails/view_helpers.rb

```diff
@@ -19,11 +19,20 @@ def mirador_tag(id: SecureRandom.uuid, height: '100%', width: '100%', position:
                "position: #{position}; display: #{display};"
       output = []
       output << content_tag(:div, '', id: id, style: styles)
-      output << '<script type="text/javascript">'       \
-                '  $(function() {'                      \
-                "    Mirador(#{config.to_json})"        \
-                '  });'                                 \
-                '</script>'
+
+      if params[:instanceName].present?
```

Comment:
Just out of curiosity, where is this `params` intended to be coming from?  In this context, (as I read it) it would be a URL parameter.  I'm just curious if you're expecting this to be one of the `options` passed into the  `mirador_tag` helper.

Reply:
Yeah. That was the idea. From what I understand of "mirador_rails" as a concept, we're trying to make Mirador a quick "tag" for rails devs, and the `options` object here has so far been intended to be Mirador options only (only things that Mirador natively recognises as a config), as opposed to quick shortcut options for the tag.

Reply:
So if they don't provide an instance name (the default is to not expect one), then no problem.

Reply:
So, if I'm using Mirador as the viewer on the detailed record/show view (not iframed or anything) and want to take advantage of this feature, I would be required to include the instanceName in my URL?  (e.g. `https://my-awesome-catalog.com/catalog/12345?instanceName=myMiradorViewer`).

That seems like something we wouldn't want to force onto our downstream consumers.

What about if we added another named parameter to the `mirador_tag` helper (which would default to `nil` so that existing consumers can opt into this behavior) and have that hold the `instance_name` that we pass through?  This seems like it would be more flexible.  If people want to pass that through a URL parameter, then they can parse that param on the view that they render the `mirador_tag` and pass it in appropriately. 

Thoughts?

Reply:
That was my intent, but I think I confused some rails controller and ruby documentation about the correct style for checking for the existence of something. This is why earlier I had used `instanceName` instead of params[:instanceName] in the interpolation, but `params[:instanceName].present?` for the branch condition.

Reply:
And similarly for the default values. I don't want to give a default value because I want it to be an optional function argument.

Reply:
`params` in the context of a helper are going to be url parameters.  That is the intent?